snaps/learn/tutorials/gas-estimation.md

```diff
@@ -211,56 +254,57 @@ To build and test your Snap:
      http://localhost:8000/
    ```
 
-3. Open [`localhost:8000`](http://localhost:8000/) in your browser (with MetaMask Flask installed).
+5.3 Open [`localhost:8000`](http://localhost:8000/) in your browser (with MetaMask Flask installed).
    A page like the following displays:
 
    <img src={require('../../assets/template-snap.png').default} alt="Test dapp with template Snap" style={{border: '1px solid #DCDCDC'}} />
 
    This is a boilerplate test dapp for installing and testing your Snap.
 
-4. Select **Connect** to connect Flask to the dapp.
+5.4 Select **Connect** to connect Flask to the dapp.
    After connecting, you're prompted to install the Snap with the following permissions:
 
-   - **Allow dapps to communicate directly with this Snap.**
+   - **Allow websites to communicate directly with this Snap.**
    - **Access the internet.**
    - **Display dialog windows in MetaMask.**
 
-5. Select **Approve** > **Install**.
+5.5 Select **Confirm** > **OK**.
 
-6. After installing, the **Send message** button on the page is enabled. Select this button. A dialog prompt displays with the response from the gas fee API:
+5.6 After installing, the **Send message** button (or **Estimate gas** button, if you followed Step 2) 
+  is enabled. Select this button. A dialog prompt displays with the response from the gas fee API:
 
 <p align="center">
 <img src={require('../../assets/gas-estimation.png').default} alt="Gas estimation dialog" width="400px" style={{border: '1px solid #DCDCDC'}} />
 </p>
 
-You have integrated a public API into MetaMask and displayed real-time gas fee estimates.
-
-### 6. Next steps
-
-Next, you can try:
-
-- Parsing the JSON response from the remote API.
-- Displaying the fees in a nicely formatted way.
+Congratulations, you have integrated a public API into MetaMask and displayed real-time gas fee estimates.
 
-You can also update the fields in `snap.manifest.json` to match your custom Snap:
+### What next?
 
-- `proposedName` - The name of your Snap.
-- `description` - The description of your Snap.
-- `source` - The `shasum` is set automatically when you build from the command line.
-  If you decided to publish your Snap to `npm`, update the `location` to its published location.
+Consider:
```

Comment:
```suggestion
To continue refining your Snap, you can complete these next steps:
```

Suggestion, can play around with wording. This is not something we do consistently in our docs, but where possible, we should avoid using fragments in introductory text for lists (for localization purposes): https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#introductory-text